Lanchonete/LanchoneteTests/Managers/ShoppingCartManagerTests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ShoppingCartManagerTests.swift
+//  LanchoneteTests
+//
+//  Created by vera.dias on 1/22/21.
+//
+
+import XCTest
+@testable import Lanchonete
+
+class ShoppingCartManagerTests: XCTestCase {
+    
+    func test_GIVEN_an_empty_shopping_cart_WHEN_I_add_a_sandwich_THEN_the_total_should_be_the_price_of_the_added_sandwich() {
+        let manager = ShoppingCartManager()
+        //let sandwich = Sandwich(type: .xBacon)
+        manager.addSandwich(type: .xBacon)
+        XCTAssertEqual(manager.getFinalPrice(), 7.0)
+    }
+    
+    func test_GIVEN_a_shopping_cart_containing_a_sandwich_WHEN_I_add_a_sandwich_THEN_the_total_should_be_the_price_of_the_sum_of_the_two_sandwiches(){
+        let manager = ShoppingCartManager()
+        
+        manager.addSandwich(type: .xBacon)
+        manager.addSandwich(type: .xBurguer)
+        XCTAssertEqual(manager.getFinalPrice(), 11.5)
```

Comment:
That's really interesting. I believe if I wanted to implement it, I would need to create a class Money that would have the unit value and cents. It's a great idea to add to the project. It will definitely be added to the backlog.  😄